docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,34 @@
+# Deploying the Node Controller Manager into a Kubernetes cluster
+
+As already mentioned, the Node Controller Manager is designed to run as controller in a Kubernetes cluster. The existing source code can be compiled and tested on a local machine as described in [Setting up a local development environment](../development/local_setup.md). Once the implementation is stabilized, you can deploy this onto your cluster using the steps described below.
+
+## Prepare the cluster
+
+- Connect to the remote kubernetes cluster where you plan to deploy the Node Controller Manager using the kubectl. Make the KUBECONFIG environment variable point to the yaml file containing your cluster info.
+- Now, create the required CRDs on the remote cluster using the following command,
+```bash
+$ kubectl apply -f kubernetes/crds.yaml
+```
+
+## Build the Docker image
+
+:warning: Modify the default docker registry with your own, modify the `Makefile` to point it to your own registry.
```

Comment:
this seems duplicate

s/Modify the default docker registry with your own, modify the `Makefile` to point it to your own registry./Modify the `Makefile` to refer to your own registry./

Reply:
Done.
https://github.com/gardener/node-controller-manager/commit/95c68a4c3f483a5f770c7de61f12b25e5411f3c0#diff-cf6b5c064a3b7eca6025325785cbbea7R15